lib/tag_fetcher.rb

```diff
@@ -0,0 +1,29 @@
+require "gds_api/content_api"
+
+class TagFetcher
+
+  def initialize(manual)
+    @manual = manual
+  end
+
+  def fetch_tags
```

Comment:
I think the convention in this app would be to call this either `call` or maybe at a push `tags`?